common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -217,15 +217,15 @@ def test_add_students_to_cohort_failure(self):
 
         self.assertEqual(
             [
-                "0 students have been added to this cohort",
-                "1 student was already in the cohort"
+                "0 learners have been added to this cohort.",
+                "1 learner was already in the cohort"
```

Comment:
@ssemenova That's fine with me. Sorry that this has come up after I provided my text, but I agree with @cahrens that "pre-registered" is misleading/confusing. Can we go with "pre-assigned" instead?